src/Modules/Helper.ts

```diff
@@ -0,0 +1,33 @@
+import Discord = require('discord.js');
+import Command from './Command';
+import KeyValueArray, { Cmd } from './KeyValue';
```

Comment:
KeyValue.ts doesn't seem to have anything called `Cmd` and the compiler is complaining about this. There is `Command` in it, though.